<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScrewCfgPage</class>
 <widget class="QWidget" name="ScrewCfgPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1326</width>
    <height>823</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,0,0,0" columnstretch="1,0,0,0,0,0">
   <item row="2" column="5" rowspan="2">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>流程</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="listWidgetWork"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnClearList">
        <property name="text">
         <string>清空流程</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>螺丝配置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4" columnstretch="1,4,3,3,0">
      <item row="0" column="0" rowspan="2">
       <widget class="QListWidget" name="screwListWidget">
        <item>
         <property name="text">
          <string>配置：1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>配置：2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>配置：3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>配置：4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>配置：5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>配置：6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>配置：7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>配置：8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>配置：9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>配置：10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QTableWidget" name="tableWidget">
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <row>
         <property name="text">
          <string>新建行</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>新建行</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>新建行</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>新建行</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>新建行</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>新建行</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>新建行</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>新建行</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>新建行</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>新建行</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>名称</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>参数</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>说明</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string>标准扭力</string>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="0" column="1">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="2">
         <property name="text">
          <string>单位N.m</string>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="1" column="0">
         <property name="text">
          <string>扭力偏差</string>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="1" column="2">
         <property name="text">
          <string>单位N.m</string>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="2" column="0">
         <property name="text">
          <string>step1圈数</string>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="2" column="2">
         <property name="text">
          <string>r</string>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="3" column="0">
         <property name="text">
          <string>step1速度</string>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="3" column="2">
         <property name="text">
          <string>r/min</string>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="4" column="0">
         <property name="text">
          <string>step2圈数</string>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="4" column="2">
         <property name="text">
          <string>r</string>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="5" column="0">
         <property name="text">
          <string>step2速度</string>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="5" column="2">
         <property name="text">
          <string>r/min</string>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="6" column="0">
         <property name="text">
          <string>step3圈数</string>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="6" column="2">
         <property name="text">
          <string>r</string>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="7" column="0">
         <property name="text">
          <string>step3速度</string>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="7" column="2">
         <property name="text">
          <string>r/min</string>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="8" column="0">
         <property name="text">
          <string>浮高圈数</string>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="8" column="2">
         <property name="text">
          <string>r</string>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="9" column="0">
         <property name="text">
          <string>滑牙圈数</string>
         </property>
        </item>
        <item row="9" column="2">
         <property name="text">
          <string>r</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QTableView" name="tableView"/>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="btnInsertScrew">
        <property name="maximumSize">
         <size>
          <width>42</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>插入</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btnSaveCfgInfo">
        <property name="text">
         <string>保存</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btnAdd">
        <property name="text">
         <string>添加</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="btnDelete">
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="5">
       <widget class="QLineEdit" name="editIoElecLockCom"/>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="checkBoxIo">
        <property name="text">
         <string>数模开关端口号</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="editIoCom"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>电批支架端口号</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="editScrewCom"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>电批端口号</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="editSupportCom"/>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="editSelectCom"/>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="checkBoxSelect">
        <property name="text">
         <string>批头选择器端口号</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QCheckBox" name="checkElecIO">
        <property name="text">
         <string>电子锁端口号</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>提示</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" columnstretch="8,8,1">
      <item row="1" column="0">
       <widget class="QPushButton" name="btnDeleteBefore">
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btnAddBefore">
        <property name="text">
         <string>添加</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QTableView" name="tipTableView"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btnInsertTips">
        <property name="maximumSize">
         <size>
          <width>42</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>插入</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>选择图片路径</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="imgPath">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="5">
    <widget class="QPushButton" name="btnCancle">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="btnConfirm">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
